src/main/resources/org/sagebionetworks/web/client/widget/discussion/SingleDiscussionThreadWidgetViewImpl.ui.xml

```diff
@@ -54,8 +54,13 @@
 					</b:Container>
 				</bh:Div>
 			</bh:Div>
-			<bh:Div ui:field="threadButtonContainer" width="98%" pull="RIGHT" addStyleNames="margin-bottom-5">
-				<b:Button ui:field="replyButton" addStyleNames="margin-top-5" icon="MAIL_REPLY">Reply</b:Button>
+			<bh:Div addStyleNames="margin-top-10" width="98%" pull="RIGHT">
+					<b:TextBox ui:field="replyTextBox" placeholder="Type your reply"/>
```

Comment:
Please change placeholder text to something like "Write a reply..." (standard ellipsis)